x=10
try:
  print(x/0)
except Exception as e:
  print(e)
